src/main/java/axoloti/utils/Preferences.java

```diff
@@ -75,6 +78,9 @@
     @Element(required = false)
     String themePath;
 
+    @Element(required = false)
+    String Platform;
```

Comment:
As long as you're making changes in Preferences, you should just change all of the fields. A capital field name is just clearly against standard Java style and shouldn't exist in the first place.